### ISSUE TITLE
arm64 support

### ISSUE THREAD

**Issue by Johennes** [YunoHost-Apps/focalboard_ynh#51]
#9 was previously closed because upstream didn't offer arm64 binaries. However, now that https://github.com/mattermost/focalboard/issues/124 has been closed there seems to be official support for this platform. Is it possible to enable the arm64 architecture for the Yunohost app now?

**Comment by ericgaspar** [comment]
yes sure, as soon as the arm64 binary is available upstream... (can't see it upstream)

**Comment by Johennes** [comment]
Yeah, I noticed as well, after opening this issue here, sorry. I've opened https://github.com/mattermost/focalboard/issues/4470 to request the binaries to be released.

**Comment by Johennes** [comment]
It looks like releasing arm64 binaries might not be a priority for them right now. Is there anything I could do in the meantime? E.g. would it help if I made and hosted arm64 builds myself?

**Comment by ericgaspar** [comment]
Yes! if you are able to compile a working binarie we could add it to the script

**Comment by Johennes** [comment]
The Dockerfile for cross-compilation is now merged. So we'd now only need to make and host the builds. Possibly we could just do this in a fork of the Focalboard repository using GitHub automation?
